### PR TITLE
add teleop gripper control

### PR DESCRIPTION
**Pull request by cretaceous-creature**
add teleop gripper control, 
now ugv and uav use some teleop source 
by different param.

For UGV teleop, published gripper topic

### CODE REVIEW

File: jsk_mbzirc_tasks/launch/jsk_mbzirc_task_2.launch [hunk 1]
```diff
@@ -14,8 +15,8 @@
   <arg name="cheat" default="false" />
 
   <!-- Spawn simulated ugv -->
-  <arg unless="$(arg cheat)" name="robot_initial_pose" value="-x 65  -y -25 -z 0.3" />
```
Comment (k-okada) [inline jsk_mbzirc_tasks/launch/jsk_mbzirc_task_2.launch]: why you change this

File: jsk_mbzirc_common/CMakeLists.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(jsk_mbzirc_common)
-
+set(CMAKE_BUILD_TYPE Release)
```
Comment (k-okada) [inline jsk_mbzirc_common/CMakeLists.txt]: why you added this?

File: jsk_mbzirc_common/src/uav_teleop_keyboard.cpp [hunk 3]
```diff
@@ -14,7 +14,7 @@
  *     * Neither the name of the <ORGANIZATION> nor the names of its
  *       contributors may be used to endorse or promote products derived from
  *       this software without specific prior written permission.
- * 
+ *
```
Comment (k-okada) [inline jsk_mbzirc_common/src/uav_teleop_keyboard.cpp]: please create separate commit for style change, add new feature, fix but..

**Comment by k-okada** [comment]
please consider updating  teleop_twist_keyboard, change this node and send PR, and use them as I do in husky_* and ur_ -> https://github.com/tongtybj/mbzirc/blob/indigo-devel/mbzirc.rosinstall

File: jsk_mbzirc_tasks/launch/jsk_mbzirc_task_2.launch (hunk 1 above)
Comment (cretaceous-creature) [inline jsk_mbzirc_tasks/launch/jsk_mbzirc_task_2.launch]: The z value is a little bit high compare to the ground, and the  then I change it to 0.1m

File: jsk_mbzirc_common/CMakeLists.txt (hunk 2 above)
Comment (cretaceous-creature) [inline jsk_mbzirc_common/CMakeLists.txt]: Sorry for that,
I use qt creator to test if I could debug... 
next time when I commit I will remove this

**Comment by cretaceous-creature** [comment]
Okada sensei,
Do you mean use teleop_twist_keyboard for both uav and 
ugv control or just use teleop_twist_keyboard for ugv control.

I am thinking of use then same key layout for control both....

**Comment by k-okada** [comment]
> Do you mean use teleop_twist_keyboard for both uav and ugv control

yes, please always think of the way not to write program code.

--
◉ Kei Okada

**Comment by cretaceous-creature** [comment]
OK I understand, thank you very much.

**Pull request merged**